libflux/flux-core/src/semantic/types.rs

```diff
@@ -494,12 +494,12 @@ impl MonoType {
     }
 }
 
-// Tvar stands for type variable.
-// A type variable holds an unknown type.
+/// A type variable holds an unknown type.
```

Comment:
This reads a little awkwardly. "A type variable _that_ holds an unknown type" maybe?

Reply:
How about:
```
/// `Tvar` stands for *type variable*.
/// A type variable holds an unknown type, before type inference.
```